packages/playwright-test/src/dispatcher.ts

```diff
@@ -215,7 +215,7 @@ export class Dispatcher {
       }));
       result.status = params.status;
       test.expectedStatus = params.expectedStatus;
-      test.annotations = params.annotations;
+      test.annotations = [...test.annotations, ...params.annotations];
```

Comment:
This will multiply annotations if we had retries.

Reply:
Updated in latest commit! Thanks!